reports/tasks.py

```diff
@@ -46,27 +46,19 @@
         day_of_week=TRIAL_INSTANCES_REPORT_SCHEDULE_DAY_OF_WEEK
     )
 )
-def send_trial_instances_report(recipients=settings.TRIAL_INSTANCES_REPORT_RECIPIENTS):
+def send_trial_instances_report(
+        recipients=settings.TRIAL_INSTANCES_REPORT_RECIPIENTS):
     """
-    Generate and send a trial instance data report
+    Generate and send a new instance data report for the past month
 
     This task runs on the first of every month at 2AM
     """
 
     if not recipients:
-        logger.warning('No recipients listed for Trial Instances Report. It will not be generated.')
+        logger.warning(
+            'No recipients listed for Trial Instances Report. It will not be generated.')
```

Comment:
@spokerman12, similar concerns as https://github.com/open-craft/opencraft/pull/711/files#r579396180

Reply:
I ran this through autopep8, but it's true that while we should adhere to PEP8 per the handbook, the 79 character per line limit is changed to 120.